lang/nl/manage-products/products.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+return [
+    "title_label" => "Producten",
+    "groups_heading" => "Groepen",
+    "actions_heading" => "Actie",
+    "action_edit_label" => "Wijzig",
+    "action_delete_label" => "Verwijder",
+    "action_edit_class" => "edit-product-button",
+    "index_page_title" => "Producten",
+    "create_page_title" => "Nieuw Product",
+    "edit_page_title" => "Wijzig Product",
+    "heading_custom_sizes" => "Maten en Prijzen",
+    "name_label" => "Naam",
+    "name_placeholder" => "Naam",
+    "price_label" => "Product prijs",
+    "price_placeholder" => "Prijs",
+    "custom_sizes_prices_label" => "Verschillende prijzen voor de maten",
+    "custom_sizes_label" => "Specifieke maten",
+    "custom_size_placeholder" => "Prijs voor maat",
+    "custom_size_button" => "Voeg maat toe",
+    "groups_multi_select_label" => "Scouting groepen",
+    "groups_multi_select_placeholder" => "Select scouting groepen",
+    "groups-multiselect" => "products-group-multiselect",
```

Comment:
?

Reply:
Gebruikt met testen en ik ben bang om dingen te slopen dus:
@adjagers zou je je code onafhankelijk hiervan willen maken? Het zit in je testen, in de views en de gloednieuwe component: multiselect